std/conv.d

```diff
@@ -3168,8 +3175,7 @@ if (isInputRange!Source && isSomeChar!(ElementType!Source) && !is(Source == enum
         // skip past the last 'f'
         ++count;
         p.popFront();
-        static if (isNarrowString!Source)
-            source = cast(Source) p;
+        advanceSource;
```

Comment:
Add `()` to function call

Reply:
Is that part of the style guide?

Reply:
It's not in [The D Style](https://dlang.org/dstyle.html), but I think de facto the style is to only omit braces for functional / property-like functions, and not omit them in 'regular' calls with side-effects.